packages/schematics/angular/ng-new/schema.json

```diff
@@ -129,9 +129,11 @@
       "x-user-analytics": 14
     },
     "strict": {
-      "description": "Creates a workspace with stricter type checking and stricter bundle budgets settings.",
+      "description": "Creates a workspace with stricter type checking and stricter bundle budgets settings. Recommended to take full advantage of TypeScript and Angular language service features.",
+      "x-prompt": "Create a workspace with stricter type checking and stricter bundle budgets settings? Recommended to take full advantage of TypeScript and Angular language service features. [https://angular.io/guide/strict-mode]",
```

Comment:
> Enforce stricter type checking and smaller bundle budgets for all projects in the workspace.
> This will improve maintainability, help catch bugs ahead of time, and allow the CLI to
> optimize your application better. To learn more, go to https://angular.io/guide/strict-mode

Language service works better with strict mode, but it shouldn't be the motivation we ask people
to enable the flag.

Reply:
That said, we could mention better experience with language service on the website.

Reply:
Amended and updated the description and prompt. Will loop in Dave just to confirm that wording.